crates/polars-time/src/chunkedarray/rolling_window/dispatch.rs

```diff
@@ -48,8 +48,7 @@ where
     let arr = ca.downcast_iter().next().unwrap();
     // "5i" is a window size of 5, e.g. fixed
     let arr = if options.window_size.parsed_int {
-        let options: RollingOptionsFixedWindow = options.into();
-        check_input(options.window_size, options.min_periods)?;
```

Comment:
I'm moving the `check_input` into `try_from`, so the `try_into` covers this (and other places where this is used won't need to remember to call `check_input`)

Reply:
Yes, good one.